.circleci/regenerate.py

```diff
@@ -49,6 +49,10 @@ def build_workflows(prefix="", upload=False, filter_branch=None, indentation=6):
                     ):
                         # the fields must match the build_docs "requires" dependency
                         fb = "/.*/"
+
+                    if os_type == "linux" and btype == "conda":
```

Comment:
We will need to keep around the py3.8 cu116 linux conda build because the docs_build job depends on that. See https://app.circleci.com/pipelines/github/pytorch/audio/13928 for reference.